plugins/src/plugins/mps/delivery_station.cpp

```diff
@@ -67,30 +63,15 @@ void DeliveryStation::new_machine_info(ConstMachine &machine)
  * reset the puck and the selected gate.
```

Comment:
Can you update the comments as well to avoid future confusions?

Reply:
Good catch! Done